lib/src/multipart_file.dart

```diff
@@ -44,20 +40,20 @@ class MultipartFile {
   /// future may be inferred from [filename].
   MultipartFile(this.field, Stream<List<int>> stream, this.length,
```

Comment:
What do you think about consolidating these constructors? I'm thinking:

* `new MultipartFile(String field, body, ...)` which takes either a `List<int>` or a `String`,
* `new MultipartFile.fromStream(String field, stream, int length)` which takes either a `Stream<List<int>>` or a `Stream<String>`,
* `static Future<MultipartFile> loadStream(String field, stream)` which takes either a  `Stream<List<int>>` or a `Stream<String>` and drains it to determine the length.

Reply:
I've always kinda preferred the explicit constructors rather than using `dynamic`  since there are no union types. I do like the `loadStream` idea rather than `fromPath`.

Would `fromStream` be possible with a `Stream<String>` though as there would need to be some kind of encoding?